website/src/content/blog/biome-v1-6.mdx

```diff
@@ -0,0 +1,273 @@
+
+<Tabs>
+    <TabItem label="Astro" icon="astro">
+        <div align="center">
+            <Image src={astroLint} alt="Screenshot of Biome linting in action for an Astro file in VSCode" width="500"  />
+        </div>
+    </TabItem>
+    <TabItem label="Vue" icon="seti:vue">
+        <div align="center">
+            <Image src={vueLint} alt="Screenshot of Biome linting in action for an Vue file in VSCode" width="500"  />
+        </div>
+    </TabItem>
+    <TabItem label="Svelte" icon="seti:svelte">
+        <div align="center">
+            <Image src={svelteLint} alt="Screenshot of Biome  linting in action for an Svelte file in VSCode" width="500"  />
+        </div>
+    </TabItem>
+</Tabs>
+
+Make sure to read the [documentation about expectations and limitations](/internals/language-support#html-super-languages-support).
+
+## Configuration, lighter and more powerful
+
+### Support for `biome.jsonc`
+
+Biome now accepts the file `biome.jsonc` as configuration! Now, you can insert all the comments you want in a configuration.
+
+### `extends` resolves dependencies
+
+From this version, Biome can use the `extends` property to *resolve* other configuration files that are inside installed dependencies.
+
+There are few important steps in order to make the configuration discoverable. The file must be exported from a `"module"` package, and the file should be exported in your `package.json` like this:
+
+```json name="shared-configs/package.json"
+{
+  "name": "@shared-configs",
+  "type": "module",
+  "exports": {
+    "./biome": "./biome.json"
+  }
+}
+```
+
+This set up allows to expose a specifier `@shared-configs/biome`, which you use inside your `biome.json` file.
+
+```json name="project/biome.json"
+{
+  "extends": ["@shared-configs/biome"]
+}
+```
+
+The resolution of the dependencies is powered by the library [`oxc-resolver`](https://github.com/oxc-project/oxc-resolver), one of the many libraries provided by the [OXC project](https://oxc-project.github.io/). It's battle-tested and spec compliant!
+
+:::note
+You can also export `biome.jsonc` files!
+
+```json name="shared-configs/package.json" ins="./biome.jsonc"
+{
+  "name": "@shared-configs",
+  "type": "module",
+  "exports": {
+    "./biome": "./biome.jsonc"
+  }
+}
+```
+:::
+
+### Reduced memory footprint
+
+We **reduced** the size our configuration **by a factor of 6.5**! This change might not have massive effects on the speed of the program, but it greatly reduced the memory used when running the CLI or the LSP.
+
+## New formatting options
+
+Other than fixes, the formatter provides two new options that should improve the compatibility with Prettier.
+
+### Option `attributePosition`
+
+The contributor [@octoshikari](https://github.com/octoshikari) implemented this new feature by themselves! Huge thank you for helping the Biome project.
```

Comment:
We could move this paragraph at the end of the section. First we explain what is the config about and then we thank the contributor. What do you think?

Reply:
Done! It makes sense